lib/src/analyzer_impl.dart

```diff
@@ -122,8 +125,13 @@ class AnalyzerImpl {
   void prepareErrors() {
     for (Source source in sources) {
       context.computeErrors(source);
+
       var sourceErrors = context.getErrors(source);
       errorInfos.add(sourceErrors);
+
+      if (options.strongMode) {
+        errorInfos.add(strongChecker.computeErrors(source));
```

Comment:
There are two options: fold the two together (whether by subclassing or by code mashing) or keep them separate and run both. Keeping them separate gives us more flexibility when designing the task structure, so I'd probably opt for that for the short term. Keeping the separate also has other performance implications because uses of the visitor pattern are fairly expensive, so we might eventually want to merge them. But like I said, at this point flexibility probably trumps performance.